tests/pretty/hir-fn-variadic.pp

```diff
@@ -9,7 +9,7 @@
 extern crate std;
 
 extern "C" {
-    fn foo(x: i32, va1: ...);
+    unsafe fn foo(x: i32, va1: ...);
```

Comment:
This is an effect of having `hir::Safety (Safe|Unsafe)`. I don't think it's worth the hassle of adding `Default` or `Safe(bool)`/`Unsafe(bool)` in order to account for things that were explicitly or implicitly marked as safe/unsafe and have or either important hir nodes growing in size or code being complicated out of something that seems what we are aiming to do long term anyway.